### PR TITLE
GET-Session1-Assignment

### PR DESCRIPTION
**Pull request by meta-nikhil-saxena**
Please review my code
In shopping cart question there are two classes :
shoppingCart.java and Product.java

### CODE REVIEW

File: PF/PF-Assignment-1/ShoppingCart/shoppingCart.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	  
+	   int choice,i=0;
+	   shoppingCart obj = new shoppingCart();
+	   
+	   do{
+			System.out.println("1. Add Product and Details");
+			System.out.println("2. Remove Product and Details");
+			System.out.println("3. Show items in the cart");
+			System.out.println("4. EXIT");
+
```
Comment (sameer-sharma) [inline PF/PF-Assignment-1/ShoppingCart/shoppingCart.java]: As per assignment -- "The items should be updatable (e.g., to change the quantity of an item in the cart,"
Current code doesn't have that. You should be able to update a item's quantity. Add an option for that.

Comment (sameer-sharma) [inline PF/PF-Assignment-1/ShoppingCart/shoppingCart.java:76]: As per assignment -- "The items should be updatable (e.g., to change the quantity of an item in the cart,"
Current code doesn't have that. You should be able to update a item's quantity. Add an option for that.


As per assignment --  "The ability to generate the bill of the cart items."
No such feature in the code. Add an option to generate total bill for cart.

**Review by sameer-sharma — CHANGES_REQUESTED** [review]

File: PF/PF-Assignment-1/ShoppingCart/shoppingCart.java (hunk 1 above)
Comment (meta-nikhil-saxena) [reply PF/PF-Assignment-1/ShoppingCart/shoppingCart.java:76]: I am still working on it...will update you as soon as I will be done....Thank you for updating the concern....

**Review by meta-nikhil-saxena — COMMENTED** [review]
I have made changes !!
Changes are in the ShoppingCart folder !!
Please review and check !!

**Review by sameer-sharma — APPROVED** [review]

**Pull request merged**